pageserver/src/tenant/timeline.rs

```diff
@@ -4344,30 +4345,39 @@ impl Timeline {
                 next_start_key: img_range.end,
```

Comment:
Regarding the debug printing above, instead of doing string formatting, we could also type much less and have:

```
debug!(
  trigger_generation,
  delta_file_accessed, // <-- this should probably be a plural?
  total_kb_retrieved,
  total_key_retrieved, // <-- plural?
  ?partition,
  "generate metadata images"
);
```

I don't understand why does it not contain the partition right now, but perhaps all metadata keys are a single partition?

It would render as does the current one but be more readable. I am however glad there is no json stringification in this one like we still have over at compaction L0 -> L1 legacy compaction.

Reply:
Yes it's a single partition